src/sql/platform/clipboard/browser/clipboardService.ts

```diff
@@ -25,6 +25,10 @@ export class BrowserClipboardService implements IClipboardService {
 		this._notificationService.info(localize('imageCopyingNotSupported', "Copying images is not supported"));
 	}
 
+	write(data: string): Promise<void> {
```

Comment:
Why are you adding this? Just use `writeText`

Reply:
This should have the same type as the vs implementation (which should be the new type you add per my suggestion below)

Reply:
And why isn't it calling vsClipboardService.write?